exercises/shared-api-gateway/ITOps.EndpointConfig/ITOps.EndpointConfig.csproj

```diff
@@ -5,7 +5,8 @@
   <ItemGroup>
     <PackageReference Include="NServiceBus" Version="7.0.1" />
     <PackageReference Include="NServiceBus.Newtonsoft.Json" Version="2.1.0" />
-    <PackageReference Include="NServiceBus.NHibernate" Version="8.0.0" />
+    <PackageReference Include="NServiceBus.Persistence.Sql" Version="4.2.0" />
+    <PackageReference Include="NServiceBus.Persistence.Sql.MsBuild" Version="4.2.0" />
```

Comment:
Does this reference have to be in this project as well as the endpoint projects themselves?

Reply:
Only in Shipping, or otherwise the saga sql for the table isn't generated.

Reply:
OK, so we can remove it from this project?